### PR TITLE
Bugzilla doc text for 4.4.12 release

### PR DESCRIPTION
**Pull request by jeana-redhat**

### CODE REVIEW

File: release_notes/ocp-4-4-release-notes.adoc [hunk 1]
```diff
@@ -3092,3 +3092,80 @@ Issued: 2020-07-06
...
+Space precluded documenting all of the container images for this release in the
+advisory. See the following article for notes on the container images in this
+release:
+
+link:https://access.redhat.com/solutions/5217641[{product-title} 4.4.12 container image list]
+
+[[ocp-4-4-12-bug-fixes]]
+==== Bug Fixes
+
+* Not blocking having several listeners, on different protocols, on the same
```
Comment (ahardin-rh) [inline release_notes/ocp-4-4-release-notes.adoc:3116]: I don't understand the first sentence. Perhaps what is meant is: "Having several listeners, on different protocols, on the same port for the `ovn-octavia` driver is blocked."

File: release_notes/ocp-4-4-release-notes.adoc [hunk 2]
```diff
@@ -3092,3 +3092,80 @@ Issued: 2020-07-06
...
+port for `ovn-octavia` driver. This was not supported previously on the
+`ovn-octavia` driver, but now it is supported and there is no need to block it.
+Several listeners on different protocols can be exposed in the same port. This
+means that it is possible to have, for instance, the DNS service to expose port
+53 in both TCP and UDP protocols when using `ovn-octavia`.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1847558[*BZ#1847558*])
+
+* Previously, the CoreDNS forward plugin used a random server selection policy
```
Comment (ahardin-rh) [inline release_notes/ocp-4-4-release-notes.adoc:3124]: a/plugin/plug-in

File: release_notes/ocp-4-4-release-notes.adoc [hunk 3]
```diff
@@ -3092,3 +3092,80 @@ Issued: 2020-07-06
...
+* During installation or upgrade, the `openshift-controller-manager` did not
+correctly report its progress condition. As a result, an installation or upgrade
+might fail. Now the Operator correctly reports its progress upon a successful
+installation or upgrade.
+(link:https://bugzilla.redhat.com/show_bug.cgi?id=1852249[*BZ#1852249*])
+
+[[ocp-4-4-12-upgrading]]
+==== Upgrading
+
+To upgrade an existing OpenShift Container Platform 4.4 cluster to this latest release, see
```
Comment (ahardin-rh) [inline release_notes/ocp-4-4-release-notes.adoc:3145]: s/OpenShift Container Platform/{product-title}

File: release_notes/ocp-4-4-release-notes.adoc [hunk 4]
```diff
@@ -3092,3 +3092,80 @@ Issued: 2020-07-06
...
+xref:../updating/updating-cluster-between-minor.html#update-upgrading-web_updating-cluster-between-minor[Updating a cluster by using the web console]
+for instructions.
+
+[IMPORTANT]
+====
+If you are upgrading to this release from {product-title} 4.3.3 or earlier, you
+must restart all Pods after the upgrade is complete.
+
+This is because the service CA is automatically rotated as of {product-title}
```
Comment (ahardin-rh) [inline release_notes/ocp-4-4-release-notes.adoc:3154]: s/This is because the service CA/The service CA

**Comment by ahardin-rh** [comment]
Just a few minor suggestions from me! :+1:

File: release_notes/ocp-4-4-release-notes.adoc (hunk 1 above)
Comment (jeana-redhat) [reply release_notes/ocp-4-4-release-notes.adoc:3116]: I didn't quite understand either but I think this works: "Previously, having several listeners on different protocols on the same port for the `ovn-octavia` driver was not supported and was blocked. With this release, it is supported and there is no need to block it."

File: release_notes/ocp-4-4-release-notes.adoc (hunk 3 above)
Comment (jeana-redhat) [reply release_notes/ocp-4-4-release-notes.adoc:3145]: fixed here and throughout the rest of this file - thanks!

File: release_notes/ocp-4-4-release-notes.adoc (hunk 4 above)
Comment (jeana-redhat) [reply release_notes/ocp-4-4-release-notes.adoc:3154]: Tabling this for now - want to take a single pass at any changes to the content of the boilerplate text later.

**Comment by ahardin-rh** [comment]
The changes LGTM! :shipit:

**Pull request merged**